clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -129,6 +129,12 @@ New checks
   Replaces certain conditional statements with equivalent calls to
   ``std::min`` or ``std::max``.
 
+- New :doc:`readability-math-missing-parentheses
```

Comment:
put this before readability-use-std-min-max

Reply:
done